aws-bundle/LICENSE

```diff
@@ -443,6 +443,12 @@ License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
 
 --------------------------------------------------------------------------------
 
+Group: software.amazon.awssdk  Name: url-connection-client  Version: 2.30.31
+Project URL: https://aws.amazon.com/sdkforjava
+License: Apache License, Version 2.0 - https://aws.amazon.com/apache2.0
+
+--------------------------------------------------------------------------------
```

Comment:
missing CRT?

Reply:
Actually, it is already present: https://github.com/apache/iceberg/blob/main/aws-bundle/LICENSE#L358